index.html

```diff
@@ -14,7 +14,7 @@
 <body>
     <div class="settings">
         <ul class="top-level-menu">
-            <li><a href="#" id="settings">Settings</a>
+            <li><a href="#" id="settings"><i class="fa fa-user-cog fa-2x" aria-hidden="true"></i></a>
                 <ul class="second-level-menu">
```

Comment:
`fa-user-cog` looks a little busy for the page. Let's use `fa-cog` instead!

Reply:
ok